src/HttpMediatR/HttpHandler.cs

```diff
@@ -17,6 +18,11 @@ public abstract class HttpHandler<TRequest> : IRequestHandler<TRequest, IActionR
     {
         private readonly ILogger _logger;
 
+        protected HttpHandler()
+        {
+            _logger = NullLogger.Instance;

```

Comment:
Oh awesome! TIL: `NullLogger`. 😄 

I figured we'd need to update the logging bits in the code to handle a potentially null `_logger` instance, but this is better/easier! 🎉